### DISCUSSION TITLE
How to set merge mode with android views

### DISCUSSION THREAD

**Discussion by csbenz** [patrykandpatrick/vico#529] [Questions] [answered]
Anybody have an example? I have the following code, but it tells me the "mergeMode" property is not found.
```

cartesianChartView.chart = CartesianChart(ColumnCartesianLayer(
                columns = listOf(
                        LineComponent(ContextCompat.getColor(context, R.color.primary)),
                        LineComponent(ContextCompat.getColor(context, R.color.secondary))
                ),
                spacingDp = 0f,
                innerSpacingDp = 0f,
//                mergeMode = ColumnCartesianLayer.MergeMode.Stacked
        ))
```

**Comment by patrickmichalik** [comment, marked as answer]
Hello! Please try replacing `mergeMode = ColumnCartesianLayer.MergeMode.Stacked` with `mergeMode = { ColumnCartesianLayer.MergeMode.Stacked }`. As can be seen [here](https://patrykandpatrick.com/vico/api/2.0.0-alpha.5/core/com.patrykandpatrick.vico.core.chart.layer/-column-cartesian-layer), `mergeMode` is a lambda with a `ColumnCartesianLayerModel` parameter, enabling you to perform dynamic `MergeMode` customization based on your chart’s data.
